package.json

```diff
@@ -227,6 +216,19 @@
     "yakbak": "^3.0.0",
     "yargs": "^16.2.0"
   },
+  "devDependencies": {
+    "@commitlint/cli": "^12.0.0",
+    "@commitlint/config-conventional": "^12.0.0",
+    "documentation": "^13.1.1",
+    "documentation-theme-utils": "^3.0.0",
+    "karma": "^6.1.1",
+    "mocha": "^8.3.0",
+    "mocha-steps": "^1.3.0",
+    "standard-changelog": "^2.0.27",
+    "standard-version": "^9.1.0",
+    "webpack-cli": "^4.5.0",
+    "webpack-dev-server": "^4.0.0-beta.0"
```

Comment:
Was the bump to a beta version to satisfy the security issues?

Reply:
Yes, but should still be compatible with Webpack 4